### DISCUSSION TITLE
What‘s recovered condition?

### DISCUSSION THREAD

**Discussion by xsmyqf** [TriliumNext/Trilium#3324] [Q&A] [answered]
I see some files tagged with recovered branch,like 
![image](https://user-images.githubusercontent.com/6007829/202629920-a5c3cf7b-1990-47f2-a21f-0752825e342e.png)
what condition will result this?

**Comment by zadam** [comment, marked as answer]
It happens when notes get "lost" in the tree - a typical example is that you have sync between work computer and personal computer, on work computer you delete note X, on personal computer you create a new child note Y under X.

When these changes get synced, X is deleted, but there's a non-deleted child Y under it. Trilium recovers child Y.

Another possibility is a bug.
